    public ProvisioningProviderException(Throwable cause) {
        super(cause);
    }

